src/internet_identity/src/state/temp_keys.rs

```diff
@@ -121,17 +107,17 @@ impl TempKeys {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq)]
-struct TempKey {
+pub struct TempKey {
```

Comment:
Is this still needed to be public?

Reply:
No, but it was like that before.
I removed it again.